hi-ml-multimodal/src/health_multimodal/image/model/pretrained.py

```diff
@@ -75,7 +75,7 @@ def get_biovil_image_encoder(pretrained: bool = True) -> ImageModel:
     return image_model
 
 
-def get_biovil_t_image_encoder() -> ImageModel:
+def get_biovil_t_image_encoder(**kwargs) -> ImageModel:
```

Comment:
Why the need for **kwargs here?

Reply:
Just so we can instantiate a classifier on top:

https://github.com/microsoft/hi-ml/blob/c61a81291288d0dd3184736f75eb9c058190904b/hi-ml-multimodal/src/health_multimodal/image/model/model.py#L57

It was necessary for one of our applications.